tables/SV-SE-G1.utb

```diff
@@ -0,0 +1,506 @@
+# liblouis: Swedish Grade 1 Braille Table
+#
+#  Copyright (C) 2022 Insyn Scandinavia AB <david@insyn.se>
+#  Copyright (C) 2012 Mesar Hameed <mesar.hameed@gmail.com>
+#  Copyright (C) 2004-2008 ViewPlus Technologies, Inc. www.viewplus.com
+#  Copyright (C) 2004-2006 JJB Software, Inc. www.jjb-software.com

```

Comment:
Could you explain in a few lines how this table came to be? It looks like you started with Se-Se-g1.utb (developed by the people in these copyright lines), and then updated it, to fix shortcomings or to match a new standard? You can maybe mention the project between Insyn Scandinavia and Punktskriftsnämnden that you mentioned in your email.

By the way you also mentioned in your email that you would not immediately replace Se-Se-g1.utb, but only remove it in a next release. I would replace it now, unless the difference between the two tables can be clearly explained. For example, maybe Se-Se-g1.utb implements an older version of the braille code that is still used. In that case we should keep Se-Se-g1.utb (and not remove it in the next Liblouis release).

Reply:
Thanks for the explanation and references.

I'll go ahead and remove Se-Se-g1.utb.